cmd/orpheus/queue_test.go

```diff
@@ -4,33 +4,80 @@ import (
 	"fmt"
 	"strconv"
 	"testing"
+	"time"
 )
 
 var tests = []struct {
```

Comment:
This isn't necessary, so I dont think we should do it, but just for reference it would probably be nicer if there was a custom struct called event which combines a list of songs, the user, and policy.

You could also probably move the song declaration to their own variables so that the tests are way more composable and readable.